Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:12
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+COPY package*.json /usr/src/app/
+RUN npm install
+COPY . /usr/src/app/
```

Comment:
you may want to add a `.dockerignore` file to exclude `node_modules`, `.git`, etc

Reply:
good catch.